src/main/java/com/xpn/xwiki/plugin/lucene/IndexUpdater.java

```diff
@@ -245,7 +245,7 @@ private void updateIndex() {
         getContext().getWiki().getStore().cleanUp(getContext());
         try {
           writer.optimize();
-          writer.close();
+          IOUtils.closeQuietly(writer);
         } catch (IOException e) {
           LOGGER.warn("Failed to close writer.", e);
```

Comment:
this warning text is irritating, because not closing the writer failed, but the optimize call.